privaterelay/migrations/0007_set_verified_email.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.7 on 2023-11-13 20:02
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("privaterelay", "0006_add_email_indexes"),
+        ("account", "0002_email_max_length"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="UPDATE account_emailaddress SET verified=True;",
+            reverse_sql="UPDATE account_emailaddress SET verified=False;",
```

Comment:
concern (non-blocking): This worries me. I know we never plan to migrate backwards, but this one seems particularly destructive? What's the effect of setting all the `account_emailaddress` values to `false` later? Are we locking everyone out?

Reply:
No, users log in by Mozilla accounts ID, not by email, so no one is locked out. Other than that, a lot of nothing happens. You'd need to verify the email before logging in by email, but we don't provide log in by email. If there is some collision between a new Mozilla account's email and an existing one, the new one will get redirected to "socialaccount_signup" (and get a `NoReverseMatch` 500), but that happens whether the email is verified or unverified.

It's mostly for database consistency. All users created before https://github.com/mozilla/fx-private-relay/releases/tag/2023.11.17 was released on November 21st have `verified=False`. All the users after have `verified=True`. If you run the reverse migration, they will all get `verified=False`, but the next new user will have `verified=True`.

Maybe someday we'll use `verified` to support multiple real emails per user. Until then, it doesn't really impact Relay at all.